api/src/application/application.service.ts

```diff
@@ -38,21 +38,21 @@ export class ApplicationService {
       .createQueryBuilder('app')
       .leftJoinAndSelect('app.assignedTo', 'assignedTo');
 
-    if (query.facilityName) {
-      queryBuilder.andWhere('app.facilityName ILIKE :facilityName', {
-        facilityName: `%${query.facilityName}%`,
+    if (query.applicationType) {
+      queryBuilder.andWhere('LOWER(app.applicationType) = LOWER(:applicationType)', {
```

Comment:
this would force the user to type the whole application type out instead of let's say `infra`, is that the intent?  I think using LIKE/ ILIKE would be better